### PR TITLE
remove project-wide deployment settings

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
There were some project wide deployment settings that were causing more harm than good given how datatools is used these days! These settings are removed in this PR

### CODE REVIEW

File: lib/manager/components/ProjectSettings.js [hunk 1]
```diff
@@ -25,66 +19,33 @@ type Props = {
...
   _updateProjectSettings = (project: Project, settings: Object) => {
-    const {updateProject} = this.props
+    const { updateProject } = this.props
     // Update project and re-fetch feeds.
     updateProject(project.id, settings, true)
   }
 
   render () {
     const {
-      activeSettingsPanel,
```
Comment (binh-dam-ibigroup) [inline lib/manager/components/ProjectSettings.js:38]: Remove this prop from type `Props`, and remove the corresponding passed prop from `ProjectViewer` line 268.

File: lib/manager/components/deployment/DeploymentSettings.js [hunk 2]
```diff
@@ -1,336 +0,0 @@
...
-}
-
-type State = {
-  buildConfig: Object,
-  routerConfig: Object,
-  useCustomOsmBounds?: boolean
-}
-
-class DeploymentSettings extends Component<Props, State> {
-  messages = getComponentMessages('DeploymentSettings')
```
Comment (binh-dam-ibigroup) [inline lib/manager/components/deployment/DeploymentSettings.js:43]: Remove the `DeploymentSettings` section from the language files (it is unused).

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Remove dead resources as indicated.

**Comment by miles-grant-ibigroup** [comment]
Good catches! Addressed

File: lib/manager/components/ProjectSettings.js [hunk 3]
```diff
@@ -25,66 +18,33 @@ type Props = {
 }
 
 export default class ProjectSettings extends Component<Props> {
-  messages = getComponentMessages('ProjectSettings')
```
Comment (binh-dam-ibigroup) [inline lib/manager/components/ProjectSettings.js:28]: Oops I just saw this one, it looks like `ProjectSettings` can be removed from the language files too!

**Review by binh-dam-ibigroup — APPROVED** [review]
Nice to see the legacy stuff go!

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM!

**Pull request merged**